Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -1936,12 +1952,15 @@ listScheduledBookings (personId, _, cityId) mbLimit mbOffset mbFromDay mbToDay m
   transporterConfig <- SCTC.findByMerchantOpCityId cityId Nothing >>= fromMaybeM (TransporterConfigNotFound cityId.getId)
   vehicle <- runInReplica $ QVehicle.findById personId >>= fromMaybeM (VehicleNotFound personId.getId)
   driverInfo <- runInReplica $ QDriverInformation.findById personId >>= fromMaybeM DriverInfoNotFound
-  driverStats <- runInReplica $ QDriverStats.findById vehicle.driverId >>= fromMaybeM DriverInfoNotFound
-  cityServiceTiers <- CQVST.findAllByMerchantOpCityId cityId
-  let availableServiceTiers = (.serviceTierType) <$> (map fst $ filter (not . snd) (selectVehicleTierForDriverWithUsageRestriction False driverStats driverInfo vehicle cityServiceTiers))
-  scheduledBookings <- runInReplica $ QBooking.findByStatusTripCatSchedulingAndMerchant mbLimit mbOffset mbFromDay mbToDay DRB.NEW mbTripCategory availableServiceTiers True cityId transporterConfig.timeDiffFromUtc
-  bookings <- mapM buildBookingAPIEntityFromBooking scheduledBookings
-  return (ScheduledBookingRes bookings)
+  case driverInfo.latestScheduledBooking of
+    Just _ -> return $ ScheduledBookingRes []
```

Comment:
We are not allowing multiple scheduled booking for driver currently, right?

Reply:
i had discussed this with PM'S they told let it be for single first after it got merged we can extend it to multiple scheduled bookings